src/Advection/upwind_biased_reconstruction.jl

```diff
@@ -117,29 +117,29 @@ const UZ{N, FT} = UpwindBiased{N, FT, <:Any, <:Any, <:Nothing} where {N, FT}
 # Uniform upwind biased reconstruction
 for buffer in advection_buffers
     @eval begin
-        @inline inner_biased_interpolate_xᶠᵃᵃ(i, j, k, grid, ::UX{$buffer, FT}, left_bias, ψ, idx, loc, args...) where FT = 
-            @inbounds ifelse(left_bias, $(calc_reconstruction_stencil(buffer, :left,  :x, false)), 
-                                        $(calc_reconstruction_stencil(buffer, :right, :x, false)))
+        @inline inner_biased_interpolate_xᶠᵃᵃ(i, j, k, grid, ::UX{$buffer, FT}, bias, ψ, idx, loc, args...) where FT = 
+            @inbounds ifelse(bias == LeftBias(), $(calc_reconstruction_stencil(buffer, :left,  :x, false)), 
+                                                 $(calc_reconstruction_stencil(buffer, :right, :x, false)))
 
-        @inline inner_biased_interpolate_xᶠᵃᵃ(i, j, k, grid, ::UX{$buffer, FT}, left_bias, ψ::Function, idx, loc, args...) where FT = 
-            @inbounds ifelse(left_bias, $(calc_reconstruction_stencil(buffer, :left,  :x, true)), 
-                                        $(calc_reconstruction_stencil(buffer, :right, :x, true)))
+        @inline inner_biased_interpolate_xᶠᵃᵃ(i, j, k, grid, ::UX{$buffer, FT}, bias, ψ::Function, idx, loc, args...) where FT = 
+            @inbounds ifelse(bias == LeftBias(), $(calc_reconstruction_stencil(buffer, :left,  :x, true)), 
```

Comment:
```suggestion
            @inbounds ifelse(bias isa LeftBias, $(calc_reconstruction_stencil(buffer, :left,  :x, true)), 
```

not a necessary change, but saves a few characters and might be easier to read